server/resolvers/deleteCollectionJobResolver.js

```diff
@@ -1,14 +1,15 @@
 const { AuthenticationError } = require('apollo-server-core');
 const {
-    deleteCollectionJob
-} = require('../models/services.deprecated/CollectionJobService');
+    removeCollectionJobById
```

Comment:
This sparks a thought on being consistent with API verb usage. So would renaming this resolver and others be helpful in the future to `remove[Entity]Resolver`?

It's outside the purpose of this PR but I'm thinking the consistency may make it easier when talking about certain files/functions as a future thought.

Reply:
Yes I definitely think that would be a win in a future PR!

Reply:
Great, let's follow that up in a separate issue!